pages/LivePage.tsx

```diff
@@ -49,6 +49,7 @@ interface UseSlotSection {
   used: boolean;
 }
 
+const useSlotSection = "$live/sections/UseSlot.tsx" as const;
```

Comment:
"UseSlotResolveType" wdyt about this name?

Reply:
Changed to `USE_SLOT_SECTION_KEY` sounds better?